src/MainWindow.vala

```diff
@@ -31,7 +31,6 @@ public class Installer.MainWindow : Gtk.Dialog {
             height_request: 700,
             icon_name: "system-os-installer",
             resizable: false,
-            title: _("Set up %s").printf (Utils.get_pretty_name ()),
```

Comment:
Normally I'd complain about removing a title entirely (since the OS uses it in other places than _just_ the window, i.e. in the dock or in some multitasking views), but since this is in the greeter… it's probably fine?

Reply:
Yeah, I would also like this to stay here, even if it's not shown currently in the window / non-existing greeter dock. This also can just say "Set up" so that we don't need to have os-release parsing.

Reply:
This is probably accurate since the scope has been greatly reduced:

```suggestion
            title: _("Create a User"),
```